### PR TITLE
Busy Man Problem in C++

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Code added in cp(competitive programming) folder.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#5072

### CODE REVIEW

**Comment by gayatri517** [comment]
@HarshCasper @ankitaggarwal23 please review my PR.

**Comment by gayatri517** [comment]
@siddharth25pandey please review my PR

File: C-Plus-Plus/cp/busy_man.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+bool compare(pair<int, int> p1, pair<int, int> p2)
+{
+    return p1.second < p2.second;
+}
+
+int main()
+{
+    int t, n, s, e;
+    cin >> t;
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/busy_man.cpp]: Use proper naming conventions don't use single letter and also while reading inputs use cout for display proper message

What is `t`? Is it used for reading testcase then remove it as we don't have online judge for testcases.

File: C-Plus-Plus/cp/busy_man.cpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        of the previous activity time */
+        if (v[i].first >= fin)
+        {
+            fin += v[i].second;
+            res++;
+        }
+    }
+
+    cout << res << endl;
+    v.clear();
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/busy_man.cpp]: Use cout for display proper message. Remove` v.clear();`

File: C-Plus-Plus/cp/busy_man.cpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Output:
+1
+2)Input:
+4
+1 7
+5 8
+7 8
+1 8
+Output:
+2
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/busy_man.cpp]: Remove extra test case one is enough

File: C-Plus-Plus/cp/busy_man.cpp [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+Time Complexity: O(nlogn)
+Space Complexity: O(n) */
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/busy_man.cpp]: Add a new line at the end

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by gayatri517** [comment]
@ankitaggarwal23 I have made the changes that were requested. Kindly review my PR.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**